packages/slate/src/models/range.js

```diff
@@ -227,7 +229,7 @@ class Range extends Record(DEFAULTS) {
    */
 
   get isSet() {
-    return this.anchorKey != null && this.focusKey != null
+    return this.anchorPoint.isSet && this.focusPoint.isSet
```

Comment:
Can you change these to be `this.anchor/focus` instead of `this.anchorPoint/focusPoint`. 

(I realize this will break `this.focus()` currently, but I will instead deprecate that in another pull request before merging this one, so that we have a migration path.)